style.css

```diff
@@ -264,16 +270,230 @@ body {
   padding-bottom: 15px;
   padding-top: 15px;
   border-radius: 10px;
-  margin-right: 20px;
+  gap: 10px;
   display: flex;
 }
 
-.m-l {
-  margin-left: 20px;
-}
-
 .language {
   display: flex;
   justify-content: space-between;
   padding-right: 60px;
 }
+
+@media only screen and (min-width: 768px) {
+  .section-a-container {
+    background-image: url("pimages/header-background-mobile.png");
+    background-position: center;
+    background-size: 100% 90%;
+    background-repeat: no-repeat;
+    display: flex;
+    padding: 0 14em;
+    justify-content: center;
+    align-items: start;
+    flex-direction: column;
+    text-align: start;
+  }
+
+  .navbar-content img {
+    display: none;
+  }
+
+  .navbar {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .nav_link {
+    display: block;
+  }
+
+  .btn {
+    margin-right: -9px;
+  }
+
+  .nav_link ul {
+    display: flex;
+    list-style: none;
+    margin-top: 50px;
+    margin-right: 50px;
+  }
+
+  .nav_link ul a {
+    text-decoration: none;
+    color: black;
+    padding: 10px;
+    font-size: 15px;
+  }
+
+  .desktop {
+    display: grid;
+    grid-template-columns: 1fr;
+    align-items: center;
+    justify-content: center;
+    background-color: #f7f7f9;
+    padding-bottom: 5%;
+  }
+
+  .container-b img {
+    width: 100%;
+    height: 100%;
+  }
+
+  .container-b {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+    gap: 2%;
+    padding: 1rem 0;
+    border-radius: 24px;
+  }
```

Comment:
The styles you applied to your desktop version are rearranging the content in the works section of your mobile version, as seen in the screenshots below:

Your design:
<img width="242" alt="image" src="https://github.com/Ejikeokegbe/Mobile-First-portifolio/assets/37116322/acc0d5c7-d185-4c7e-b082-679561d838e6">

Figma design:
<img width="214" alt="image" src="https://github.com/Ejikeokegbe/Mobile-First-portifolio/assets/37116322/5f5db438-4af2-47c3-9de8-6a6e8b2a68ec">

Please make your that your mobile version is unaffected by your desktop styling🙏